packages/typegpu/src/tgpuBindGroupLayout.ts

```diff
@@ -136,6 +136,11 @@ export interface TgpuBindGroupLayoutExperimental<
     TgpuLayoutEntry | null
   >,
 > extends TgpuBindGroupLayout<Entries> {
+  /**
+   * Marks a bind group layout to appear with a specific group index in the resulting WGSL code
+   */
```

Comment:
Tiny nit, we might be able to rephrase this to also note on the automatic indexing if the method is not called.

```suggestion
  /**
   * Associates this bind group layout with an explicit numeric index. When a call to this
   * method is omitted, a unique numeric index is assigned to it automatically.
   *
   * Used when generating WGSL code: `@group(${index}) @binding(...) ...;`
   */
```